dispatcher/dispatcher.go

```diff
@@ -79,7 +86,7 @@ func (d *Dispatcher) dispatchBatch(ctx context.Context, events []*entities.Event
 		return d.db.Attempts.BatchInsert(ctx, attempts)
 	})
 	if err == nil {
-		go d.sendToQueue(context.TODO(), attempts)
+		go d.sendToQueue(context.WithoutCancel(ctx), attempts)
```

Comment:
Is it safe to share the same context? What if the ctx is canceled when the client request ends abnormally?

Reply:
WithoutCancel returns a copy of parent that is not canceled when parent is canceled